README.md

```diff
@@ -228,236 +250,52 @@ messages about the conversion.
-extra characters are all spaces.
-
-### Storage Use
-
-The tool maps several PostgreSQL types to Spanner types that use more storage.
-For example, `SMALLINT` is a two-byte integer, but it maps to Spanner's `INT64`,
-an eight-byte integer. This additional storage could be significant for large
-arrays.
-
-### Arrays
-
-Spanner does not support multi-dimensional arrays. So while `TEXT[4]` maps to
-`ARRAY<STRING(MAX)>` and `REAL ARRAY` maps to `ARRAY<FLOAT64>`, `TEXT[][]` maps
-to `STRING(MAX)`.
-
-Also note that PosgreSQL supports array limits, but the PostgreSQL
-implementation ignores them. Spanner does not support array size limits, but
-since they have no effect anyway, the tool just drops them.
-
-### Primary Keys
-
-Spanner requires primary keys for all tables. PostgreSQL recommends the use of
-primary keys for all tables, but does not enforce this. When converting a table
-without a primary key, HarbourBridge will create a new primary key of type
-INT64. By default, the name of the new column is `synth_id`. If there is already
-a column with that name, then a variation is used to avoid collisions.
-
-### NOT NULL Constraints
-
-The tool preserves `NOT NULL` constraints. Note that Spanner does not require
-primary key columns to be `NOT NULL`. However, in PostgreSQL, a primary key is a
-combination of `NOT NULL` and `UNIQUE`, and so primary key columns from
-PostgreSQL will be mapped to Spanner columns that are both primary keys and `NOT
-NULL`.
-
-### Foreign Keys and Default Values
-
-Spanner does not currently support foreign keys or default values. We drop these
-PostgreSQL features during conversion.
-
-### Other PostgreSQL features
-
-PostgreSQL has many other features we haven't discussed, including functions,
-sequences, procecdures, triggers, (non-primary) indexes and views. The tool does
-not support these and the relevant statements are dropped during schema
-conversion.
-
-See
-[Migrating from PostgreSQL to Cloud Spanner](https://cloud.google.com/spanner/docs/migrating-postgres-spanner)
-for a general discussion of PostgreSQL to Spanner migration issues.
-HarbourBridge follows most of the recommendations in that guide. The main
-difference is that we map a few more types to `STRING(MAX)`.
+Please refer below mentioned links for details on schema conversion of PostgreSQL
+and MySQL database:
+
+- [PostgreSQL schema conversion](postgres/README.md#schema-conversion)
+- [MySQL schema conversion](mysql/README.md#schema-conversion)
 
 ## Data Conversion
 
-HarbourBridge converts PostgreSQL data to Spanner data based on the Spanner
+HarbourBridge converts PostgreSQL/MySQL data to Spanner data based on the Spanner
 schema it constructs. Conversion for most data types is fairly straightforward,
 but several types deserve discussion. Note that HarbourBridge is not intended
-for databases larger than a couple of GB.
+for databases larger than a couple of GB. Please refer below mentioned links for details on
```

Comment:
Replace "Please refer below mentioned links" with "Please refer to the links below".